src/hal/i_components/time.icomp

```diff
@@ -83,7 +83,7 @@ function _;
 
 FUNCTION(_)
 {
-    __u32 totalseconds;
+    hal_u32_t totalseconds;
     if(start && !old_start) totalnsec = 0;
```

Comment:
old_start changed to hal_bit_t, 0 to 0.0 and some pedantic typecasting to (hal_u32_t) in next refresh